src/ApiService/ApiService/OneFuzzTypes/Validated.cs

```diff
@@ -63,10 +61,14 @@ public record PoolName : ValidatedString {
     }
 
     public static bool TryParse(string input, [NotNullWhen(returnValue: true)] out PoolName? result) {
-        if (!Check.IsAlnumDash(input)) {
-            result = default;
-            return false;
-        }
+
+        // bypassing the validation because this code has a stricter validation than the python equivalent
```

Comment:
@Porges  FYI, I had to disable this validation because is stricter that the python code. 
In python you can create a pool names "pool_linux" (with underscore). which should not work see #2080. But with this code, The pool name cannot be retrieved from the DB.